internal_packages/keybase/lib/passphrase-popover.cjsx

```diff
@@ -1,25 +1,67 @@
 {React, Actions} = require 'nylas-exports'
 Identity = require './identity'
+PGPKeyStore = require './pgp-key-store'
 _ = require 'underscore'
+fs = require 'fs'
+pgp = require 'kbpgp'
 
 module.exports =
 class PassphrasePopover extends React.Component
   constructor: ->
-    @state = {passphrase: ""}
+    @state = {
+      passphrase: ""
+      placeholder: "PGP private key password"
+      classNames: "key-passphrase-input form-control"
```

Comment:
Typically  you want React component state to reflect the conceptual state of the component, rather than what that means in terms of HTML text / classes / styles. I think this will work, but it'd be cleaner to say `@state = {error: false}` and then in the render method, add an extra className if `error = true`.